httpserver/swarm.cpp

```diff
@@ -160,7 +160,7 @@ static all_swarms_t apply_ips(const all_swarms_t& swarms_to_keep,
             if (other_snode_it != other_snode_map.end()) {
                 const auto& other_snode = other_snode_it->second;
                 // Keep swarms_to_keep but don't overwrite with default IPs
-                if (snode.ip() == "0.0.0.0") {
+                if (other_snode.ip() != "0.0.0.0") {
```

Comment:
This condition should also check that the IP is different than the current value (otherwise `updates_count` will be entirely meaningless).  Perhaps:
```C++
auto& new_ip = other_snode_it->second.ip();
if (new_ip != "0.0.0.0" && snode.ip() != new_ip) {
  snode.set_ip(new_ip);
  updates_count++;
}
```

Reply:
Updated.